manifests/s/soopercool101/BrawlCrate/0.40a/soopercool101.BrawlCrate.installer.yaml

```
@@ -0,0 +1,18 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: soopercool101.BrawlCrate
+PackageVersion: 0.40a
+InstallerType: exe
+InstallerSwitches:
+  Silent: /silent
+  SilentWithProgress: /silent
+  Custom: ''

```

```suggestion
  Silent: -y
  SilentWithProgress: -y
```
-y to hide the prompt, it will however just extract all the files to your download folder